src/@utils/orbis.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactNode } from 'react'
+import Link from 'next/link'
+import reactStringReplace from 'react-string-replace'
+
+/** Regex patterns to use */
+const patternMentions = /\B@[a-z0-9_.⍙-]+/gi
+
+export function didToAddress(did: string) {
```

Comment:
we can apply the suggestion above with types to utils methods as well, and have more `_utils.tsx` where needed if the utils from the file can't be used in more than one component.

see '@context/Prices' for eg